src/connector/src/connector_common/common.rs

```diff
@@ -683,6 +685,8 @@ impl NatsCommon {
         let consumer = stream
             .get_or_create_consumer(&name, {
                 config.deliver_policy = deliver_policy;
+                config.durable_name = Some(durable_consumer_name);
+                config.filter_subjects = self.subject.split(',').map(|s| s.to_string()).collect();
                 config
```

Comment:
Doc link: https://docs.rs/async-nats/latest/async_nats/jetstream/consumer/pull/struct.Config.html#structfield.durable_name

For filter_subjects, the doc said this, should we care about it?:
```
filter_subjects:
Available on crate feature server_2_10 only.
```

Reply:
`server_2_10` is included as the default feature. And they do not mention requirement on the server side.